core/vmpu/inc/vmpu.h

```diff
@@ -161,8 +161,8 @@ extern int vmpu_fault_recovery_bus(uint32_t pc, uint32_t sp, uint32_t fault_addr
 
 uint32_t vmpu_fault_find_acl(uint32_t fault_addr, uint32_t size);
 
-extern void vmpu_acl_stack(uint8_t box_id, uint32_t bss_size, uint32_t stack_size, uint32_t * bss_start,
-                           uint32_t * stack_pointer);
+extern void vmpu_acl_sram(uint8_t box_id, uint32_t bss_size, uint32_t stack_size, uint32_t * bss_start,
```

Comment:
This was actually just creating an ACL for the box private stack. Historically we only had a stack, then a small context was added, then the whole BSS sections. The name was mistakenly kept (so nothing to do with *stacking* stuff).

I agree with the "functions as verb" approach, but I would like to not change this specific one now. This would require fixing other functions in the file and in other vMPU drivers. I'd like to fix them when we introduce the v8-M driver.